pymc/distributions/timeseries.py

```diff
@@ -452,7 +465,13 @@ def dist(
                     f"got ndim_supp={init_dist.owner.op.ndim_supp}.",
                 )
         else:
-            init_dist = Normal.dist(0, 1, size=(*sigma.shape, ar_order))
+            warnings.warn(
+                "Initial distribution not specified, defaulting to "
+                "`Normal.dist(0, 100, shape=...)`. You can specify an init_dist "
+                "manually to suppress this warning.",
+                UserWarning,
+            )
+            init_dist = Normal.dist(0, 100, size=(*sigma.shape, ar_order))
```

Comment:
Should use shape.

Reply:
+1 for picking one or the other.

Changing it